modules/ai-agents/pages/mcp-server/quickstart.adoc

```diff
@@ -351,10 +395,16 @@ claude mcp add local -- npx mcp-remote http://localhost:4196/sse
 
 == Next steps
 
-Try adding more tools under the same `example` tag to expand Claude Code's capabilities. See xref:ai-agents:mcp-server/developer-guide.adoc[].
+Now that you understand how MCP works with Redpanda Connect, you can create custom tools for your specific workflows:
+
```

Comment:
@JakeSCahill The em dashes in every sentence here is cringy. I'd edit this whole bolded list and then make the 2 extra items at the end of the generated list look like they fit in better.